pysatMissions/plot.py

```diff
@@ -72,107 +66,28 @@ def plot_simulated_data(inst, filename=None):
     ax3.axes.get_xaxis().set_visible(False)
     plt.legend(loc=(1.01, 0.15))
     ax3.xaxis.set_major_formatter(matplotlib.dates.DateFormatter('%H:%M'))
-    # # xlabels = [label[0:6] for label in xlabels]
-    # plt.setp(ax3.xaxis.get_majorticklabels(), rotation=20, ha='right')
 
-    ax4 = f.add_subplot(gs[3, 0], sharex=ax)
+    ax4 = fig.add_subplot(gs[3, 0], sharex=ax)
     plt.plot(inst['B_sc_x']*1e5, color='b', linestyle='--')
     plt.plot(inst['B_sc_y']*1e5, color='r', linestyle='--')
     plt.plot(inst['B_sc_z']*1e5, color='g', linestyle='--')
     ax4.set_title('Magnetic Field in S/C X, Y, and Z')
     ax4.set_ylabel('Gauss')
     ax4.set_ylim([-3.5, 3.5])
-    # ax3.axes.get_xaxis().set_visible(False)
     plt.legend(loc=(1.01, 0.15))
     ax4.xaxis.set_major_formatter(matplotlib.dates.DateFormatter('%H:%M'))
-    # # xlabels = [label[0:6] for label in xlabels]
     plt.setp(ax4.xaxis.get_majorticklabels(), rotation=20, ha='right')
 
-    # inst info
-    ax6 = f.add_subplot(gs[4, 0])
-
-    # do world plot if time to be plotted is less than 285 minutes, less than
-    # 3 orbits
-    time_diff = inst.data.index[-1] - inst.data.index[0]
-    if time_diff > pds.Timedelta(minutes=285):
```

Comment:
Loss of code/features always makes me sad.

Reply:
Since basemap has gone away, there's not much of a point of this.  I have a cartopy version somewhere, but this does not play well with travis.

Reply:
I'm sure you've seen this, but this looks like not that many lines that could get travisCI working with cartopy:
https://laszukdawid.com/2017/06/04/installing-cartopy-on-ubuntu-14-04-or-travis-ci/

I haven't tried it though so it might be a dead end. Especially since I don't think this will work for all of the OS virtual machines that Travis uses.

Reply:
Thanks!